dev-requirements.txt

```diff
@@ -1,7 +1,7 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@jerco/convert-caching-tests#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@jerco/convert-caching-tests#egg=dbt-tests-adapter&subdirectory=tests/adapter
```

Comment:
TODO: change back after https://github.com/dbt-labs/dbt-core/pull/6612 is merged, before merging this PR